py-polars/polars/interchange/buffer.py

```diff
@@ -52,10 +52,10 @@ def bufsize(self) -> int:
             n_bits = offset + length
             n_bytes, rest = divmod(n_bits, 8)
             # Round up to the nearest byte
-            if rest:
-                return n_bytes + 1
-            else:
+            if rest == 0:
                 return n_bytes
+            else:
+                return n_bytes + 1
```

Comment:
The logic here is the same, but I had to re-implement the same logic on the Rust side and this more closely mirrors that implementation.